ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java

```diff
@@ -33,12 +32,12 @@
     @Before
     public void setup() {
         patient = createTestPatient();
-        createTestVisit();
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
```

Comment:
this method does the same thing below so no need of this change

Reply:
Thanks @k-joseph for this! Am updating soonthe pr with the recommendations